File: pinyougou/pinyougou-manage-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    //查询所有数据
      this.findAll=function () {
          return $http.get("../brand/testPage.do");
      }

      //分页查询
      this.findPage=function () {
          return  $http.get("../brand/testAll.do?page=" + page + "&rows=" + rows);
      }

      //添加
      this.add=function (entity) {
          return $http.post("../brand/add.do",entity);
      }

      //修改
      this.update=function (entity) {
          return $http.post("../brand/update.do",entity);
      }


    //根据id查询
      this.findone=function () {
          return  $http.get("../brand/findone.do?id="+id);
      }

      //删除
      this.delete=function (arrayids) {
          return $http.get("../brand/delete.do?ids="+arrayids);
      }

    //搜索
     this.search =function (searchEntity,page,rows) {
         return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
     }
})

File: pinyougou/pinyougou-sellergoods/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.PageResult;

import java.util.List;

public interface BrandService extends BaseService<TbBrand> {

    public List<TbBrand>  queryAll();

 List<TbBrand> testpage(Integer page, Integer rows);

 PageResult search(TbBrand brand, Integer page, Integer rows);
}
